pom.xml

```diff
@@ -251,6 +251,12 @@
 				<version>${netty.version}</version>
 				<optional>true</optional>
 			</dependency>
+			<dependency>
+				<groupId>javax.servlet</groupId>
+				<artifactId>javax.servlet-api</artifactId>
+				<version>${javax.servlet-api.version}</version>
+				<optional>true</optional>
```

Comment:
No, just import doesn't mean anything. Cookie class would be loaded (and fail) if somebody actually would use these new methods.